data/hai/hai reveal 4 middle.txt

```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	You're still finishing up a little shortly after they leave, at which point a harried-looking Sayari appears and practically sprints across the room to you.`
-				goto alarm
-			label dinner
-			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the 'Elders' Chambers' - which turns out to be the official name of the whole government complex.`
-			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. I mean, seriously, Tee, what will you do if it doesn't? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
-			`	Teeneep just sips her drink and shakes her head. "Oh, Sayari, you really are getting too long in the tooth for domestic politics these days."`
-			`	A moment later Sayari squeaks and looks up in horror. "You triggered an election?! This wasn't just making it public, you had the petition for redress ready to go all along!"`
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hoped would happen, isn't it?"`
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
+				`	"You can trust the Unfettered to defend you, but you must be careful to avoid a war with the humans. That would be catastrophic for everyone."`
+				`	"The Unfettered have offered their help because you would not help your people. If you change your decision, you will not need their involvement."`
+					goto "turn back unfettered"
+			action
+				set "told hai to accept unfettered help"
+			`	One Elder rises and says, "The Unfettered only know how to make war. Once they are in human space, there's no telling what they'll do." Many in the room nod in agreement.`
+			`	Teeneep responds, "They are clever strategists and dedicated in our protection. Never would they take such unnecessary risks. I am certain they would accept an overseer among us to advise them on limits to their actions."`
+				goto "the elders vote"
+			label "turn back unfettered"
+			action
+				set "told hai to reject unfettered help"
+			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
+			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. Fighting at Wah Ki has stopped, as the military is anticipating an agreement."`
```

Comment:
I think that if fighting stopped at wah ki we need to make it actually stop trough an event

Reply:
Can you please find where the code is that stops the fighting?

I don't know where it happens, but it isn't in this mission. It's earlier.

Reply:
Oh so it actually happens? Because the old text said it had intensified or smth?

Reply:
It is possible I removed it by accident. I'll replay the Hai Reveal and make sure it's still there.

Reply:
I played trough HR I'm pretty sure it did not happen

Reply:
It definitely should happen, as the rest of Hai Reveal doesn't make sense without it.

Reply:
I don't want to touch that in this PR though. Filling that plot hole would require a code change in the Wanderers campaign too. Instead, I'll just remove Teeneep's statement.

Reply:
Idk if it would require changes in Wand honestly
Yeah also they say it intensifies since the start but it didnt change one bit bruh

Could be for another PR for sure

Reply:
The change in the Wanderers campaign is to restore the hostility. In other words, the Hai Reveal would change the affinity so the Hai factions are friends, but Sayari's visit will change them back to enemies.

That definitely does *not* belong in this PR though, so I'll just modify the dialog so it doesn't need a new government affinity.

Reply:
After re-reading the Hai Reveal/Wanderers campaign connections, I see some bigger plot holes. Somehow the enormous political clout Teeneep had mysteriously never existed in the Wanderers campaign. She also seems to be hugely pro-Unfettered, not caring much about the Wanderers, even though she put a lot of work into keeping peace with the humans during their dealings with the Unfettered. This is easy to fix without substantive changes to the Wanderers campaign, but I'll work on that after 7586 has more review.

Reply:
> The change in the Wanderers campaign is to restore the hostility. In other words, the Hai Reveal would change the affinity so the Hai factions are friends, but Sayari's visit will change them back to enemies.
> 
> That definitely does _not_ belong in this PR though, so I'll just modify the dialog so it doesn't need a new government affinity.

Oh it makes them friendly I thought it stopped the incoming ships
I saw a new gov "friendly uhai" was made so I thought that wasnt the case

Reply:
No, that's only for the ships the Hai allow to pass through their territory. I'm 83% certain all other Unfettered fleets are in the original government.

Reply:
Teeneeps prior mention of Unfettered not attacking is now replaced with a warning: the Fettered probably could not stop the entire Unfettered armada, and it is unnecessary bloodshed anyway.